internal/venice-common/src/main/java/com/linkedin/venice/controllerapi/ControllerClient.java

```diff
@@ -994,19 +995,30 @@ public VersionResponse getStoreLargestUsedVersion(String clusterName, String sto
     return request(ControllerRoute.GET_STORE_LARGEST_USED_VERSION, params, VersionResponse.class);
   }
 
-  public RegionPushDetailsResponse getRegionPushDetails(String storeName, String clusterName) {
-    QueryParams params = newParams().add(CLUSTER, clusterName).add(NAME, storeName);
+  public RegionPushDetailsResponse getRegionPushDetails(
+      String storeName,
+      String clusterName,
```

Comment:
Same here. clusterName can be removed since ControllerClient already knows the cluster name.

Reply:
Thanks and will fix it.